src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3405,6 +3405,8 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
                         is_inline = true;
                     } else if (name == "static") {
                         is_static = true;
+                    } else if (name == "jit") {
+                        is_jit = true;
```

Comment:
I think `jit` should not compile with LPython, but let's raise some informative message, like "@lpython.jit` decorator must be run from CPython, not compiled using LPython".